types/backend_url.pp

```diff
@@ -0,0 +1,5 @@
+type Sensu::Backend_URL = Variant[
```

Comment:
It can be useful to comment regex's with what they should match, such as  the following. This makes it easy to grok what's happening.

`Sensu Backend URL's use ws:// or wss:// or can be specified with the host only and no protocol. They must always have the port specified.`

Reply:
So `ws://` or `wss://` are actually always required but made optional because they will be automatically added by `sensu::agent` class based on value of `use_ssl`.  I am making note of that in comment.